packages/amqp/package.json

```diff
@@ -37,7 +37,7 @@
         "amqplib": "^0.10.8"
     },
     "devDependencies": {
-        "@biomejs/biome": "2.0.5",
+        "@biomejs/biome": "^1.9.4",
```

Comment:
Switching back to biome `1.9.4` as the new version is not compatible with our config in `lokalise/biome-config`. We need to upgrade there first. (The version in MQT was updated automatically few days ago, that's why we didn't have issues before: https://github.com/kibertoad/message-queue-toolkit/pull/296/files)